connect_sensor.sh

```diff
@@ -45,14 +45,26 @@
 		echo "Now attempting to download the support library.  If this script exits it's because it was unable to download the library with curl or wget." >&2
 		if [ ! -r ./acmlib.sh ]; then
 			if type -path curl >/dev/null 2>&1 ; then
-				curl -fsSL -O https://raw.githubusercontent.com/activecm/shell-lib/master/acmlib.sh || exit 1
+				curl -fsSL -O https://raw.githubusercontent.com/activecm/shell-lib/master/acmlib.sh
```

Comment:
Please add the `--fail` flag to this curl command. Otherwise, when curl gets back 4xx it will save the 4xx document to the target file and return 0. .... Learned this one the hard way over on the BeaKer project.